configs/webpack-config-compass/src/webpack-plugin-start-electron.ts

```diff
@@ -67,8 +67,10 @@ export class WebpackPluginStartElectron {
       const rendererEntries = Object.keys(this.rendererCompiler.options.entry);
       const rendererOutputPath = this.rendererCompiler.options.output
         .path as string;
-      const devServerPort: number =
-        this.rendererCompiler.options.devServer?.port ?? 4242;
+
+      const devServerPort: number = this.rendererCompiler.options.devServer
+        ? this.rendererCompiler.options.devServer.port
```

Comment:
This changes the logic and we're not accounting for port property missing on the devServer params object, is this intentional?

Reply:
so according to webpack types, the devserver is now:
```
/**
* Options for the webpack-dev-server.
*/
devServer?: false | { [index: string]: any };
```
it used to be undefined instead of false, so optional chaining doesn't work anymore. I don't think we set the "no devServer" explicitly, so it's just this reading that has to be changed.

Reply:
ah yeah, my mistake 🤦 thanks for noticing!